ui/app/styles/core/buttons.scss

```diff
@@ -347,6 +347,27 @@ a.button.disabled {
     color: $white;
   }
 
+  &.hidden-icon svg {
+    color: transparent;
+  }
+
+  &.size-7 div {
+    font-size: $size-7;
+  }
+
+  &.code-snippet {
+    background: none;
+    border: none;
+    box-shadow: none;
+    min-width: auto;
+    padding: $size-11 $size-8;
+    color: $ui-gray-300;
+
+    svg {
+      color: $ui-gray-300;
+    }
+  }
+
   &.is-primary {
```

Comment:
Considering changing this name to `primary` instead (and the other to `secondary`) so as not to be confused by the `is-primary` class nested under the `button` class. [@kiannaquach  brought up in my last PR](https://github.com/hashicorp/vault/pull/22614#discussion_r1317635182) that the 2 `is-primary` classes were confusing at first until seeing that one is nested under `button` class and the other is nested under `hds-copy-button` class. (More details as to why I chose to create this new class can be seen in that thread of comments).

Thoughts?

Reply:
I think keeping the class names as `is-primary` and `is-secondary` makes sense since it follows our current class patterns. In your handoff, I would have a screenshot of a the primary + secondary buttons and how to use the class :)

Reply:
Sounds good! I'll add that info to the handoff, too!